docs/source/index.rst

```diff
@@ -1,10 +1,39 @@
 Welcome to legendhpges's documentation!
-==========================================
+=======================================
 
-Table of Contents
------------------
+Python package of Germanium detector geometries for radiation transport simulations.
+
+Getting started
+---------------
+
+legendhpges can be installed with pip.
+
+.. code-block:: console
```

Comment:
Shouldn't we advise users to directly install from PyPI, or to do an editable install (`pip install **-e**`)? With the current instructions, they have a code folder that they cannot edit, but might expect to be able to do so...